.gitignore

```diff
@@ -31,3 +31,7 @@ fuzz-incremental/
 # cargo configuration. We presently use this to create custom cargo profiles
 # that should not be checked in at this location.
 .cargo/
+LOCK
+db/
```

Comment:
if we add a same name folder like src/raft, can it be ignored too?

Reply:
@siddontang If it's better, I can qualify the patterns to be at the root, like `/raft/` instead.

Reply:
@siddontang I changed the PR to qualify the ignore patterns as being in the root, f.ex. /LOCK.